R/gsutil.R

```diff
@@ -102,16 +102,36 @@ NULL
     stop("failed to find 'gsutil' binary")
 }
 
+## Get gsutil billing project
+.gsutil_billing_project <-
+    function()
+{
+    ## Get gcloud binary
+    gcloud <- .gsutil_find_binary('gcloud')
+    args <- c('config', 'list', 'project')
+    res <- system2(gcloud, args, stdout = TRUE, stderr=TRUE)[2]
```

Comment:
More robust to ask for `--format=json` and use `jsonlite::fromJSON()$core$project` ?

Need to check for warnings / errors.